linodecli/baked/operation.py

```diff
@@ -781,29 +790,43 @@ def parse_args(self, args: Any) -> argparse.Namespace:
         :rtype: Namespace
         """
 
-        #  build an argparse
-        parser = argparse.ArgumentParser(
+        parser, list_items = self.build_parser()
+
+        parsed = parser.parse_args(args)
+
+        if self.method in ("post", "put"):
+            self._validate_parent_child_conflicts(parsed)
+
+        return self._handle_list_items(list_items, parsed)
+
+    def build_parser(
```

Comment:
I split this into multiple functions so we can extract the generated usage text for use in the docs